.github/workflows/test.yaml

```diff
@@ -870,7 +870,7 @@ jobs:
           # propagated; and we use a subshell as this option could trigger
           # undesired changes elsewhere
           echo "Running command: $SPREAD $RUN_TESTS"
-          (set -o pipefail; $SPREAD -no-debug-output -logs .logs $RUN_TESTS | ./tests/lib/external/snapd-testing-tools/utils/log-filter $FILTER_PARAMS | tee spread.log)
+          (set -o pipefail; $SPREAD -no-debug-output -logs .logs $RUN_TESTS | PYTHONDONTWRITEBYTECODE=1 ./tests/lib/external/snapd-testing-tools/utils/log-filter $FILTER_PARAMS | tee spread.log)
```

Comment:
Perhaps we could just save this env var in a previous step as we do in "name: Prepare test results env and vars"

echo "PYTHONDONTWRITEBYTECODE=1" >> $GITHUB_ENV

Reply:
I wanted to be precise here but I think we could actually drop this patch entirely, as we are also protected by the changes in spread.yaml and in the prepare script.